include/jemalloc/internal/jemalloc_internal.h.in

```diff
@@ -145,6 +145,13 @@ static const bool have_thp =
     false
 #endif
     ;
+static const bool have_getcpu =
```

Comment:
Quick question: do we still want a mallctl to introspect have_percpu_arena? If so, do we want to call it "config_have_percpu_arena"? We have a mallctl for opt_percpu_arena which returns the current mode. I checked have_dss and didn't seem to find one that returns have_dss directly.